build.gradle

```diff
@@ -1,6 +1,6 @@
 buildscript {
     ext {
-        osisVersion = '2.1.3'
+        osisVersion = '2.1.4'
```

Comment:
this is completely unrelated to the move the ghcr, may be worth pointing it out in the PR description & title